server/leafnode.go

```diff
@@ -74,6 +74,11 @@ func (c *client) isSolicitedLeafNode() bool {
 	return c.kind == LEAF && c.leaf.remote != nil
 }
 
+// Returns true if this is a solicited leafnode and is not configured to be treated as a hub.
+func (c *client) isSpokeLeafNode() bool {
+	return c.kind == LEAF && c.leaf.remote != nil && !c.leaf.remote.Hub
```

Comment:
Why is Hub public?

Reply:
It is an option of `RemoteLeafOpts` that `leafNodeCfg` embeds. I could duplicate and move it in `leafNodeCfg`, but I see that we use Credentials, etc.. directly, so I figured it was ok.

Reply:
ok sounds good.